### DISCUSSION TITLE
Iframe / Video

### DISCUSSION THREAD

**Discussion by RudyBekker** [markdoc/markdoc#328] [Q&A] [answered]
Hey guys, I am trying to embed a loom.com video via iFrame and not succeeding at all! how should I approach embedding iframes?

**Comment by mfix-stripe** [comment, marked as answer]
Hey @RudyBekker, we created a YouTube embed component for our Docs here: 
* https://github.com/markdoc/docs/blob/main/markdoc/tags/youtube.markdoc.js
* https://github.com/markdoc/docs/blob/main/components/YouTube.js

Does that help?

**Reply by RudyBekker** [reply]
Yes it does! thanks a milly guys!
